game/theater/conflicttheater.py

```diff
@@ -365,7 +365,7 @@ def add_preset_locations(self) -> None:
         for group in self.ewrs:
             closest, distance = self.objective_info(group)
             if distance < self.BASE_DEFENSE_RADIUS:
-                closest.preset_locations.ewrs.append(
+                closest.preset_locations.base_air_defense.append(
```

Comment:
This is going to break quite a few campaigns and it doesn't have to. Previously (2.4) EWRs could be placed explicitly within the base. Since these are smallish single units instead of huge SA-10 sites, they can be put very close to the runway without issue. Adding those locations to the list of potential SAM locations means SAMs spawning on the runway.

Instead, add a `base_ewrs` to the `PresetLocations` class.